Corona-Warn-App/src/test/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiarySettingsTest.kt

```diff
@@ -0,0 +1,39 @@
+package de.rki.coronawarnapp.contactdiary.ui
+
+import de.rki.coronawarnapp.util.preferences.FlowPreference
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class ContactDiarySettingsTest {
+
+    @MockK lateinit var preferences: ContactDiaryPreferences
+    @MockK lateinit var intPreference: FlowPreference<Int>
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+        every { preferences.onboardingStatusOrder } returns intPreference
+    }
+
+    @Test
+    fun `not onboarded`() {
+        every { intPreference.value } returns -1
```

Comment:
Instead of the hardcoded `-1`, you could return `ContactDiarySettings.OnboardingStatus.NOT_ONBOARDED.ordinal` here.

Reply:
Not a fan, because it creates a dependency to an entity of the domain it is supposed to persist, but if it is your Christmas wish...

Reply:
:) Makes sense. Nope, leave it as it is then.